test/test_loader.py

```diff
@@ -44,8 +45,9 @@ def test_startup_shutdown(self):
         loader.shutdown()
         self.assertFalse(os.path.exists(pidfile))
 
-    def test_load_all(self):
-        """Check that load_records is called with the right arguments."""
+    def test_load_all_empty(self):
+        """Check that load_records is called with an empty list."""
+        logger = logging.getLogger('loader')
```

Comment:
Why's the contents of this test changed?

Reply:
It now also tests that logging is called correctly for that case (in addition to the existing assert) and I altered the description to clarify the differences between load_all tests.